devices/profalux.js

```diff
@@ -0,0 +1,91 @@
+const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const utils = require('zigbee-herdsman-converters/lib/utils');
+const reporting = require('zigbee-herdsman-converters/lib/reporting');
+const e = exposes.presets;
+const ea = exposes.access;
+
+const tzCover = {
```

Comment:
This looks very similar to https://github.com/Koenkk/zigbee-herdsman-converters/blob/325a960821dba3597cb510246a34aa39dbff8a7f/converters/toZigbee.js#L296 , could you use the existing converter instead?

Reply:
Hum, cover_via_brightness do not allow to send the STOP command.
Any hint to refactor this code?

Reply:
I can add a condition in cover_via_brightness to handle the stop command and use it if you preferer?

Reply:
> I can add a condition in cover_via_brightness to handle the stop command and use it if you preferer?

yes please